frontend/pages/Homepage/cards/MDM/MDM.tsx

```diff
@@ -84,13 +108,20 @@ const MDM = ({
         },
         { status: "Unenrolled", hosts: unenrolled_hosts_count },
       ]);
+      setSolutions(mobile_device_management_solution);
```

Comment:
I think you can just `setSolutions(Object.values(mobile_device_management_solution));` which then allows you to eliminate the `generateSolutionsDataSet` function altogether (see also related comment on the table config file below).